deployer/hub.py

```diff
@@ -550,16 +554,6 @@ def deploy(self, auth_provider, secret_key, skip_hub_health_test=False):
 
         generated_values = self.get_generated_config(auth_provider, secret_key)
 
-        # Ensure helm charts are up to date
-        helm_charts_dir = (Path(__file__).parent.parent).joinpath("helm-charts")
-        subprocess.check_call(
-            ["helm", "dep", "up", helm_charts_dir.joinpath("basehub")]
-        )
-        if self.spec["helm_chart"] == "daskhub":
-            subprocess.check_call(
-                ["helm", "dep", "up", helm_charts_dir.joinpath("daskhub")]
-            )
-
```

Comment:
I would strongly prefer us not to remove this block. It was added because the tech team kept running into issues that needed a run of `helm dep up` locally and we kept 'forgetting', leading to lost time trying to debug when this was all we needed to run. There are also no steps in CI currently to run this step, so this removal equates to "never run `helm dep up` ever".

Therefore I would like us to keep this block here for now and open a discussion on where to put it in the future (if this is not the correct place).

refs: https://github.com/2i2c-org/infrastructure/issues/556 and https://github.com/2i2c-org/infrastructure/issues/716

Reply:
Ignore this comment - I just found where you moved it to!